### PR TITLE
feat: Confirm password added

### PR DESCRIPTION
**Pull request by Niharika0104**
## Related Issue

Confirm Password is missing

## Description
I have implemented a confirmation password feature on the frontend, and the validation processes are functioning correctly.

## Screenshots

fixed

### CODE REVIEW

**Comment by sumanjeet0012** [comment]
hey @Niharika0104 ,
The GitHub expects title of a PR in a more systematic way.
checkout https://www.conventionalcommits.org/en/v1.0.0/#summary

As you have implemented a new feature you can write like 
feat: Confirm password added

After correcting your PR title the `title needs formatting` label will get removed from your PR.

File: .env.example [hunk 1]
```diff
@@ -3,4 +3,5 @@
 # NEXT_PUBLIC_USER_COLLECTION="Appwrite Cloud user collection ID"
 # NEXT_PUBLIC_POSTS_COLLECTION="Appwrite Cloud post collection ID"
 # NEXT_PUBLIC_BUCKET_ID="Appwrite Cloud bucket ID"
-# NEXT_PUBLIC_BASE_URL="base url of the web app (e.g. localhost:3000)"
\ No newline at end of file
+# NEXT_PUBLIC_BASE_URL="base url of the web app (e.g. localhost:3000)"
+# NEXT_PUBLIC_BASE_URL="base url of the web app (e.g. localhost:3000)"
```
Comment (Sanchitbajaj02) [inline .env.example:7]: Why extra line is added? I think it is not required, you should remove it

File: package-lock.json
Comment (Sanchitbajaj02) [inline package-lock.json:1]: Please remove package.json since it will give conflicts in production

File: package.json [hunk 2]
```diff
@@ -21,21 +21,23 @@
...
     "postcss": "8.4.31",
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-feather": "^2.0.10",
     "react-redux": "^8.1.2",
     "react-toastify": "^9.1.3",
     "tailwindcss": "3.3.3",
-    "typescript": "5.1.6"
+    "typescript": "5.1.6",
+    "yarn": "^1.22.21"
```
Comment (Sanchitbajaj02) [inline package.json:40]: yarn should be a global dependency and not be added into the project. Please remove it as well

File: src/backend/appwrite.config.ts [hunk 3]
```diff
@@ -2,7 +2,7 @@
 import { Account, Client, Databases, Storage, ID, Query } from "appwrite";
 
 const appwriteClient = new Client();
-
+//https://cloud.appwrite.io/v1
```
Comment (Sanchitbajaj02) [inline src/backend/appwrite.config.ts:5]: please remove this comment. Please don't put unnecessary comments.

File: src/components/pages/auth/register/index.tsx
Comment (Sanchitbajaj02) [inline src/components/pages/auth/register/index.tsx:1]: Where is the logic which verify that both the passwords entered are exactly same?

**Review by Sanchitbajaj02 — CHANGES_REQUESTED** [review]

File: src/backend/appwrite.config.ts (hunk 3 above)
Comment (Niharika0104) [reply src/backend/appwrite.config.ts]: hey i resolved all the issues(cheked whether both the passwords are matching or not ,removed package.json file ,removed unnecessary comments,password is atleast of length 8 , contains atleast one digit ,atleast one alphabet ,
atleast one special character.I used regex for checking password strength.
I checked these things on the frontend.I didnot check it backend wise.
If u still find any issues let me know.

**Review by Sanchitbajaj02 — APPROVED** [review]

**Pull request merged**